omnibus_build.sh

```diff
@@ -22,9 +22,20 @@ rm -rf /opt/$PROJECT_NAME/*
 cd $PROJECT_DIR
 # Allow to use a different dd-agent-omnibus branch
 git fetch --all
-git checkout $OMNIBUS_BRANCH 
+git checkout $OMNIBUS_BRANCH
 git reset --hard origin/$OMNIBUS_BRANCH
 
+# If an RPM_SIGNING_PASSPHRASE has been passed, let's import the signing key
+if [ -n "$RPM_SIGNING_PASSPHRASE" ]; then
+  gpg --import /keys/RPM-SIGNING-KEY.private
+fi
+
+# Last but not least, let's make sure that we rebuild the agent everytime because
+# the extra package files are destroyed when the build container stops (we have
+# to tweak omnibus-git-cache directly for that). Same for gohai.
+git --git-dir=/var/cache/omnibus/cache/git_cache/opt/datadog-agent tag -d `git --git-dir=/var/cache/omnibus/cache/git_cache/opt/datadog-agent tag -l | grep datadog-agent`
```

Comment:
datadog-agent and datadog-gohai should have the always_build flag set to true, It's not enough ?